Gemfile.lock

```diff
@@ -140,10 +141,11 @@ PLATFORMS
 DEPENDENCIES
   jbundler (~> 0.9.2)
   jdbc-mariadb (~> 1.1)
+  jdbc-postgres (~> 9.4)
   jdbc-sqlite3 (~> 3.7)
   killbill-cybersource!
   rake (>= 10.0.0)
   rspec (~> 2.12.0)
 
 BUNDLED WITH
-   1.11.2
+   1.13.6
```

Comment:
Bundler version needs to be kept at 1.11.2 (as set by our official build environment).